parsl/executors/status_handling.py

```diff
@@ -80,6 +95,20 @@ def bad_state_is_set(self):
     def executor_exception(self):
         return self._executor_exception
 
+    @property
+    def error_management_enabled(self):
+        return True
+
+    def handle_errors(self, error_handler: "parsl.dataflow.job_error_handler.JobErrorHandler",
+                      status: Dict[Any, JobStatus]) -> bool:
+        init_blocks = 3
```

Comment:
It's a bit harder to read when there's some logic here in handle_errors and some over in the `simple_error_handler`, 
when this could just be consolidated in the JobErrorHandler.